.teamcity/Gradle_Check/model/PerformanceTestBucketProvider.kt

```diff
@@ -305,17 +305,15 @@ echo "Performance tests to be ${action}d in this build"
 cat $performanceTestSplitDirectoryName/$action-$fileNamePostfix
 """
 
-    val linesWithEcho = csvLines.joinToString("\n") { "echo $it" }
+    val linesWithEcho = csvLines.joinToString("\n") { """echo $it >> $performanceTestSplitDirectoryName\$action-$fileNamePostfix""" }
 
     val windowsScript = """
 mkdir $performanceTestSplitDirectoryName
 del /f /q $performanceTestSplitDirectoryName\include-$fileNamePostfix
 del /f /q $performanceTestSplitDirectoryName\exclude-$fileNamePostfix
-(
 $linesWithEcho
-) > $performanceTestSplitDirectoryName\$action-$fileNamePostfix
 
-echo "Performance tests to be ${action}d in this build"
+echo Performance tests to be ${action}d in this build
```

Comment:
Not sure how important this is but it would probably be nice to instead properly escape the quotes in `${action}`.

Reply:
This is not about escaping quotes, since there are no quotes in `action`. The output of the command `echo "Performance tests to be ${action}d in this build"` is `"Performance tests to be ${action}d in this build"` while the output of the command `echo Performance tests to be ${action}d in this build` is `Performance tests to be ${action}d in this build`. The quotes are printed out by `echo` on Windows and they are unnecessary.